Hearthstone Deck Tracker/Utility/Helper.cs

```diff
@@ -54,8 +54,8 @@ public static class Helper
 		public static readonly Dictionary<string, string> LanguageDict = new Dictionary<string, string>
 		{
 			{"English", "enUS"},
-			{"Chinese (China)", "zhCN"},
-			{"Chinese (Taiwan)", "zhTW"},
+			{"简体中文", "zhCN"},
+			{"繁體中文", "zhTW"},
```

Comment:
There should be a better way of doing it (e.g. using `CultureInfo.GetCultureInfo(...).NativeName`), but I think translating this might be fine for now.